docs/sources/mimir/references/architecture/components/compactor/index.md

```diff
@@ -126,11 +126,20 @@ The storage retention is disabled by default, and no data will be deleted from t
 
 For more information, refer to [Configure metrics storage retention]({{< relref "../../../../configure/configure-metrics-storage-retention" >}}).
 
-## Compactor disk utilization
+## Compactor scratch storage volume
+
+Each compactor uses a storage device mounted at `-compactor.data-dir` to temporarily store:
+
+- files downloaded from object storage used as input to compaction
+- block files produced by the compactor to be uploaded to object storage
 
-The compactor needs to download blocks from the bucket to the local disk, and the compactor needs to store compacted blocks to the local disk before uploading them to the bucket. The largest tenants may need a lot of disk space.
+{{% admonition type="note" %}}
+We recommend using a volume other than the root volume to avoid I/O contention with the operating system. Common volume types include a local SSD or a cloud provider's block storage service. Either of these can be mounted with a Kubernetes `PersistentVolumeClaim`. While the compactor is a stateless service, we run compactors in a Kubernetes `StatefulSet` so that each pod gets a dedicated volume.
+{{% /admonition %}}
```

Comment:
- Although it's not consistent in these docs in particular, in general, the documentation should avoid first person pronouns like "we".
- For Kubernetes objects, we follow the same guidelines as in the Kubernetes docs (https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/#kubernetes-objects), so you don't need to put these resources in inline code. I've also capitalized Pod.
- The other changes try to use active voice over passive voice.
- I've also tried to rewrite the note so that it isn't specific to Kubernetes (and the bit that is, says so. 
 
Let me know what you think, I'm not sure that I've nailed the rewrite so I'm very happy to iterate on that.

```suggestion
{{% admonition type="note" %}}
While the compactor is a stateless service, it's recommended that you configure the compactor to store its temporary files somewhere other than the root volume.
Common volume types include a local SSD or a cloud provider's block storage service.
This avoids I/O contention with the operating system.  

In Kubernetes, run compactors as a StatefulSet so that each Pod has a dedicated volume.
{{% /admonition %}}
```